packages/block-library/src/button/index.js

```diff
@@ -33,6 +33,8 @@ export const settings = {
 	supports: {
 		align: true,
 		alignWide: false,
+		reusable: false,
+		html: false,
```

Comment:
It looks like this change affects mobile, so was wondering what the best way to update this setting on mobile would be (cc @pinarol)?

Is it best to keep the mobile setting as it currently is ... e.g.:

```js
export const settings = {
	...webSettings,
	supports: {
		align: true,
		alignWide: false,
	},
	parent: undefined,
};
```